src/main/resources/assets/xp_storage/lang/pt_br.json

```
@@ -0,0 +1,38 @@
+{
+  "item.xp_storage.crystallized_lapis": "Lápis-lazúli cristalizado",
+  "item.xp_storage.xp_book_upgrade_smithing_template": "Molde de ferraria",
+  "item.xp_storage.smithing_template.xp_book_upgrade.additions_slot_description": "Adicione diamante, lingote de netherita ou estrela do nether",
+  "item.xp_storage.smithing_template.xp_book_upgrade.applies_to": "Livro de xp",
+  "item.xp_storage.smithing_template.xp_book_upgrade.base_slot_description": "Adicione um livro de xp",
+  "item.xp_storage.smithing_template.xp_book_upgrade.ingredients": "Diamante, lingote de netherita e estrela do nether",
+  "item.xp_storage.xp_book": "Livro de xp I",
```

```suggestion
  "item.xp_storage.xp_book": "Livro de xp",
```

Only one XP Book will remain, the other ones are being deprecated. There is no need of an explicit number anymore, this is now shown in the tooltip (`"item.xp_storage.xp_bottle.tooltip.level"`)